tests/rocprof-sys-videodecode-tests.cmake

```
@@ -0,0 +1,34 @@
+# -------------------------------------------------------------------------------------- #
+#
+# video decode tests
+#
+# -------------------------------------------------------------------------------------- #
+
+rocprofiler_systems_add_test(
+    SKIP_BASELINE SKIP_RUNTIME
+    NAME videodecode
+    TARGET videodecode
+    RUN_ARGS -i ${ROCmVersion_DIR}/share/rocdecode/video/ -t 2
+    REWRITE_ARGS
+        -e
+        -v
+        2
+        -R
+        run
+        --allow-overlapping
+        --print-available
+        functions
+        --print-overlapping
+        functions
+        --print-excluded
+        functions
+        --print-instrumented
+        functions
+        --print-instructions
+    LABELS "videodecode")
```

```suggestion
rocprofiler_systems_add_test(
    SKIP_BASELINE SKIP_RUNTIME SKIP_REWRITE
    NAME videodecode
    TARGET videodecode
    GPU ON
    RUN_ARGS -i ${ROCmVersion_DIR}/share/rocdecode/video/ -t 2
    LABELS "videodecode")
```
Added `GPU ON`, so test will check for a valid GPU before running. 
Also, consider simplifying the test for this iteration, since we're just testing for the VCN Activity track. For the moment, we can add `SKIP_RUNTIME` to save on testing time.